scripts/run-delta-sweep.py

```
@@ -0,0 +1,175 @@
+                        "TotalWrites"))
+    sep = "-" * len(header)
+
+    outf = None
+    if args.output:
+        outf = open(args.output, "w", newline="\n")
+
+    csvf = None
+    csvw = None
+    if args.csv:
+        csvf = open(args.csv, "w", newline="")
+        fieldnames = ["depth", "table", "total_writes"] + BIN_NAMES
```

Ruff (0.15.5)

[warning] Consider `["depth", "table", "total_writes", *BIN_NAMES]` instead of concatenation

Replace with `["depth", "table", "total_writes", *BIN_NAMES]`

(RUF005)